README.md

```diff
@@ -11,12 +11,25 @@ jobs:
   steps:
     ...
     - name: Upload wheel
-      uses: scientific-python/upload-nightly-action@main
+      uses: scientific-python/upload-nightly-action@8f0394fd2aa0c85d7364a9958652e8994e06b23c # 0.1.0
       with:
         artifacts_path: dist
         anaconda_nightly_upload_token: ${{secrets.UPLOAD_TOKEN}}
 ```
 
+It is recommended that [Dependabot][] is used to keep the GitHub Action updated
+to the latest release by using a `.github/dependabot.yml` config file similar to
+
+```yaml
```

Comment:
can we link to a docs instead keeping this example here? E.g. I wonder about maintenance when e.g. `version: 2` becomes outdated, etc.

Reply:
> can we link to a docs instead keeping this example here?

Sure! @henryiii just migrated https://scikit-hep.org/developer/gha_basic#updating to https://learn.scientific-python.org/development/guides/gha_basic/#updating so we can use the shiny new Scientific Python docs for this. :+1:

Though I would both link it _and_ keep it in the README as it seemed from Issue #8 that making things easy to copy and paste for adoption was something that while not an explicit goal would be the recommendation.

> I wonder about maintenance when e.g. version: 2 becomes outdated, etc.

I'll let @henryiii correct me but I believe this is a very slow release cycle so I think the answer is that the maintenance burden is minimal.

Reply:
OK, let's remove it if/when it becomes an issue.